### ISSUE TITLE
paper_radio_button_basic_test 'button respects the user set aria-label' failing

### ISSUE THREAD

**Issue by ErikGrimes** [dart-lang/polymer_elements#42]
```
expect(c.attributes['aria-label'], equals("Batman"));
```

```
 Expected: 'Batman'
    Actual: 'Robin'
     Which: is different.
  Expected: Batman
    Actual: Robin
            ^
   Differ at offset 0

  package:test                             expect
  paper_radio_button_basic_test.dart 73:7  main.<async>.<fn>.<fn>
```

**Comment by jakemac53** [comment]
This actually is passing for me, not sure what changed but I did update the elements recently so maybe that's it.
